<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addoredit_srez</class>
 <widget class="QDialog" name="addoredit_srez">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>добавление записи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resourses.qrc">
    <normaloff>:/img/img/paper.png</normaloff>:/img/img/paper.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ОК</string>
     </property>
     <property name="icon">
      <iconset resource="../resourses.qrc">
       <normaloff>:/img/img/ok.png</normaloff>:/img/img/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="icon">
      <iconset resource="../resourses.qrc">
       <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_error">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>Заполните обязательные поля</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите группу</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_gruppa">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите дату проведения</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QDateEdit" name="lineEdit_god">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="displayFormat">
         <string>dd MMMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дисциплину 1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QComboBox" name="comboBox_discipline_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дисциплину 2</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QComboBox" name="comboBox_discipline_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дисциплину 3</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QComboBox" name="comboBox_discipline_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дисциплину 4</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QComboBox" name="comboBox_discipline_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите дисциплину 5</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QComboBox" name="comboBox_discipline_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>comboBox_discipline_1</zorder>
     <zorder>label_4</zorder>
     <zorder>label_error</zorder>
     <zorder>label_6</zorder>
     <zorder>comboBox_gruppa</zorder>
     <zorder>lineEdit_god</zorder>
     <zorder>comboBox_discipline_2</zorder>
     <zorder>label_2</zorder>
     <zorder>comboBox_discipline_3</zorder>
     <zorder>label_3</zorder>
     <zorder>comboBox_discipline_4</zorder>
     <zorder>label_5</zorder>
     <zorder>comboBox_discipline_5</zorder>
     <zorder>label_7</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
